pkg/testutil/recorder.go

```diff
@@ -92,12 +92,13 @@ func (r *recorderStream) Record(a Action) {
 }
 
 func (r *recorderStream) Action() (acts []Action) {
+out:
```

Comment:
or you can set a var like finished, and set finished to true to break for loop. try to avoid the tag anyway.